docs/DE0001.md

```diff
@@ -6,26 +6,28 @@ T:System.Security.SecureString
 
 ## Motivation
 
-* The purpose of `SecureString` is to avoid having secrets stored in the process
+* The purpose of [`SecureString`](SecureString) is to avoid having secrets stored in the process
```

Comment:
You need to use angle brackets for the link, not parenthesis as you're pointing to the URL below. Otherwise the link doesn't work.

Reply:
But the right one would be square brackets, not angle, correct?

Reply:
Correct, I said angel brackets and I meant square. Already edited my comment to hide my sins, but you noticed.